lib/chef/knife/core/generic_presenter.rb

```diff
@@ -1,6 +1,6 @@
 #--
 # Author:: Daniel DeLeo (<dan@chef.io>)
-# Copyright:: Copyright 2011-2016, Chef Software, Inc.
```

Comment:
your script went off the rails a bit here

Reply:
no, i got sick of the years, time for them to go away.

Reply:
We should just nuke them all then